### ISSUE TITLE
[Feature Request]: Excluding img tags from color contrast rule check

### ISSUE THREAD

**Issue by csaszika** [storybookjs/storybook#22813]
### Is your feature request related to a problem? Please describe

I have a slider component and because of business reasons I have to show it with real images which violate the color-contrast rule, because I cannot change them. We fetch those images from content management system.

### Describe the solution you'd like

Turn off color-contrast rule for img tags.

### Describe alternatives you've considered

Currently we turn off color contrast rule for the story, but this slider could have a "title" which should be checked with color contrast rule too.

### Are you able to assist to bring the feature to reality?

no

**Comment by liwn9527** [comment]
hi @csaszika , could you provide a example in codesandbox or others?
